### PR TITLE
Added `.xcprivacy` for Xcode 15

### PR DESCRIPTION
**Pull request by NachoSoto**
Example generated report:

Screenshot 2023-06-09 at 15 16 11

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -36,13 +36,22 @@ let package = Package(
     targets: [
         .target(name: "RevenueCat",
                 path: "Sources",
-                exclude: ["Info.plist", "LocalReceiptParsing/ReceiptParser-only-files"]),
+                exclude: ["Info.plist", "LocalReceiptParsing/ReceiptParser-only-files"],
+                resources: [
+                    .copy("../Sources/PrivacyInfo.xcprivacy")
```
Comment (NachoSoto) [inline Package.swift:41]: This kind of breaks (cc @neonacho)
Screenshot 2023-06-09 at 15 40 13

/cc @

**Comment by jefago** [comment]
Here are our related docs. It really is a bit tricky because a lot depends on how the developer is using RevenueCat. Theoretically we should just mark "Purchase history" as being collected: https://www.revenuecat.com/docs/apple-app-privacy

**Comment by NachoSoto** [comment]
Okay just updated it to only purchase history.

File: Package.swift (hunk 1 above)
Comment (NachoSoto) [reply Package.swift:41]: I'll leave this for now since it's the only way to add it.

**Review by jefago — APPROVED** [review]

File: Package.swift [hunk 2]
```diff
@@ -36,13 +36,22 @@ let package = Package(
...
                 exclude: ["Info.plist", "LocalReceiptParsing/ReceiptParser-only-files"],
+                resources: [
+                    .copy("PrivacyInfo.xcprivacy")
+                ],
                 swiftSettings: [.define("ENABLE_CUSTOM_ENTITLEMENT_COMPUTATION")]),
         .target(name: "ReceiptParser",
-                path: "LocalReceiptParsing"),
+                path: "LocalReceiptParsing",
+                resources: [
+                    .copy("../Sources/PrivacyInfo.xcprivacy")
```
Comment (tonidero) [inline Package.swift:53]: Hmm does the receipt parser also use UserIds?

**Review by tonidero — APPROVED** [review]

File: Package.swift (hunk 2 above)
Comment (NachoSoto) [reply Package.swift:53]: Oops good catch, let me remove it from this one.

**Pull request merged**